src/features/components/templates/footer/data.tsx

```diff
@@ -498,91 +499,119 @@ export const socialButtonsROW = [
     {
         icon: <StandaloneFacebookIcon iconSize="md" fill="#000000b8" />,
         href: 'https://www.facebook.com/derivdotcom',
-        'data-cy': 'facebook',
-        'aria-label': 'facebook',
+        'data-cy': SocialProviders.Facebook,
+        'aria-label': SocialProviders.Facebook,
     },
     {
         icon: <StandaloneInstagramIcon iconSize="md" fill="#000000b8" />,
         href: 'https://www.instagram.com/deriv_official',
-        'data-cy': 'instagram',
-        'aria-label': 'instagram',
+        'data-cy': SocialProviders.Instagram,
+        'aria-label': SocialProviders.Instagram,
     },
     {
         icon: <StandaloneXTwitterIcon iconSize="md" fill="#000000b8" />,
         href: 'https://twitter.com/derivdotcom/',
-        'data-cy': 'twitter',
-        'aria-label': 'twitter',
+        'data-cy': SocialProviders.Twitter,
+        'aria-label': SocialProviders.Twitter,
```

Comment:
Why do we need to change these? if it's a need Please help to fix the test failure also in cypress due to changes in data-cy